.gitignore

```diff
@@ -61,3 +61,6 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+# IDE
+.idea
```

Comment:
Rather use a local ignore file for this. This is not necessary be ignored for the repository.

There are 3 types to ignore files:

1. Push .gitignore file in repo (globally for the whole repository for all developers)
2. Add a .git/info/exclude file inside your local repository (just locally for just this repository)
3. Use global git config (ignore files globally on your local machine for all repositories)

Reply:
Have a look here: https://help.github.com/articles/ignoring-files/